vendor-extra/LiberoPageBundle/src/Resources/translations/messages+intl-icu.en.xlf

```diff
@@ -24,6 +24,10 @@
                 <source>No items available.</source>
             </trans-unit>
 
+            <trans-unit id="5" resname="libero.page.info_bar.demo">
+                <source>This is a demo site.</source>
```

Comment:
Thinking we have a problem if we want to introduce a link...

Reply:
Currently HTML will be escaped. Either we find a hacky syntax here and process the results, or raw HTML is allowed (ideally in just specific cases).

Reply:
The latter is definitely simple (https://github.com/libero/browser/pull/92/commits/75af57ebb93aa867a8142b48a2b9e4464c191f19).